plutus-metatheory/src/Scoped.lagda

```diff
@@ -32,17 +32,13 @@ open import RawU using (TyTag;TmCon;tmCon;tagCon2TmCon;tmCon2TagCon)
 \end{code}
 
 \begin{code}
-data ScopedTy (n : ℕ) : Set
-
-import Builtin.Constant.Type ℕ ScopedTy as S
-
-data ScopedTy n where
+data ScopedTy (n : ℕ) : Set where
   `    : Fin n → ScopedTy n
   _⇒_  : ScopedTy n → ScopedTy n → ScopedTy n
   Π    : Kind → ScopedTy (suc n) → ScopedTy n
   ƛ    : Kind → ScopedTy (suc n) → ScopedTy n
   _·_  : ScopedTy n → ScopedTy n → ScopedTy n
-  con  : S.TyCon n → ScopedTy n
+  con  : ∀{K} → TyCon K → ScopedTy n
   μ    : ScopedTy n → ScopedTy n → ScopedTy n
   missing : ScopedTy n -- for when things compute to error
```

Comment:
Would be great to have an explanation why we need that `missing` thing.

Reply:
I don't think it is needed anymore. I'll try and remove it.